opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -203,6 +203,8 @@ namespace Opm {
     BlackoilWellModel<TypeTag>::
     linearize(SparseMatrixAdapter& jacobian, GlobalEqVector& res)
     {
+        GlobalEqVector res_local;
```

Comment:
Consider making this a member to avoid reallocation every linearize() call.

Reply:
Fixed.